openapi.yaml

```diff
@@ -5289,6 +5289,15 @@
           "schema" : {
             "type" : "string"
           }
+        }, {
+          "name" : "numberOfDays",
+          "in" : "query",
+          "required" : false,
+          "schema" : {
+            "type" : "integer",
```

Comment:
Can this be an arbitrary number, or should it be an enum? Or a range?

Reply:
it can be any arbitrary number although I haven't yet but was considering adding a 90 day max check on the server side

Reply:
Also maybe guard against negative integers ^^